source/includes/_all-get-apis.md

```diff
@@ -22,128 +22,128 @@ https: // *actsapi.intelex.com/API-develop/v1/AttributeType?PageNumber=1&PageSiz
 
 All data from the Compound table will be returned from the endpoint below. You can optionally fetch specific data by including compound ID’s, compound Type IDs, compound status IDs, compound Names, external identifiers, or CAS number. The Compound endpoint supports pagination.
 
-> Example GET Endpoint Without Pagination: 
+> Example Of Compound GET Endpoint Without Pagination: 
 
 ```
-https: // *actsapi.intelex.com/API-develop/v1/Compound
+https: // {tenant}actsapi.intelex.com/API-develop/v1/Compound
 ```
 
-> Example GET Endpoint With Pagination: 
+> Example Of Compound GET Endpoint With Pagination and Filter Option: 
 
 ```
-https: // *actsapi.intelex.com/API-develop/v1/Compound? PageNumber=1&PageSize=500&compoundIDs="int data"&compoundTypeIDs="int data"&compoundstatusIDs="int data"&compoundNames="string data"&externalIdentifier="int"&CAS="string data"
+https: // {tenant}actsapi.intelex.com/API-develop/v1/Compound? PageNumber=1&PageSize=500&compoundIDs="int data"&compoundTypeIDs="int data"&compoundstatusIDs="int data"&compoundNames="string data"&externalIdentifier="int"&CAS="string data"
 ```
 
 ## 3.Emission Category API GET Endpoint 
 
 All data from the Emission Category table will be returned from the endpoint below. You can optionally fetch specific data by including the emission category ID. The Emission Category endpoint supports pagination.
 
 
-> Example GET Endpoint Without Pagination: 
+> Example Of EmissionCategory GET Endpoint Without Pagination: 
 
 ```
-https: // *actsapi.intelex.com/API-develop/v1/EmissionCategory
+https: // {tenant}actsapi.intelex.com/API-develop/v1/EmissionCategory
```

Comment:
is it `API` or `API-develop`? Review other URLs as well

Reply:
it's API-develop  here develop is the environment mentioned here , the final URLs would be changed in the Doc once we finalize the URL and we get the prod environment .

Reply:
The final URL's would be something different and would be modified in the GA release .